docs/explore.md

```diff
@@ -26,16 +26,16 @@ This is an explicit non-goal of the project.
 
 There is a tension in programming language design between scalability and safety.
 To provide temporal memory safety (no use after free bugs), you need to know when an object will never be accessed again.
-This is however, typically, a global problem as the accesses can be across many threads and determining when an object is no longer accessed requires some form of consensus between the threads.
-This is what a global GC does with its stop the world phases, which scan memory to determine if something is unreachable.
+This is, however, typically a global problem as the accesses can be across many threads and determining when an object is no longer accessed requires some form of consensus between the threads.
+This is what a global GC does with its stop-the-world phases, which scan memory to determine if something is unreachable.
 This dynamic consensus that an object is no longer in use can cause 
 either latency issues or throughput.
 There is amazing research in doing this better
 but in Verona we want to take an alternative approach: ownership.
 
-Ownership has been used a in a few languages to support scalable memory management.
+Ownership has been used in a few languages to support scalable memory management.
 Rather than allowing multiple threads to access a single object, we ensure that at most one thread can access an object at a time.
-This is fundamental in languages like [Pony](https://www.ponylang.io) and [Rust](https://www.rust-lang.org).
+This is fundamental in languages like [Pony](www.ponylang.io) and [Rust](https://www.rust-lang.org).
```

Comment:
Looks like you edit is reverting #28.

Reply:
Oops, sorry and thank you for notifying this.